dashboard/app/mailers/ops_mailer.rb

```diff
@@ -19,4 +19,13 @@ def script_assigned(params)
     subject = "You have been assigned a new course: #{params[:script].localized_title}"
     mail content_type: 'text/html', subject: subject, to: params[:user].email
   end
+
+  def unexpected_teacher_added(workshop_facilitator, added_teachers, workshop)
```

Comment:
my ocd would say maybe just use the name facilitator everywhere in this method instead of workshop_facilitator